### PR TITLE
Add system information

### PR DESCRIPTION
**Pull request by danielfrg**
Added some info about the system, cleaned up the UI a bit and added packrat.

Let me know what you think.

### CODE REVIEW

**Comment by colearendt** [comment]
Oops!! Sorry @danielfrg ! I totally never saw this haha. Definitely want to merge this, but I suspect we should use `renv` at this point 😅 Happy to take on refactoring if you'd rather me handle it! 

I also generally like to remove `.Rprofile` / `.gitignore` it so that users can opt into or out of the restore process

**Comment by danielfrg** [comment]
Yes sir you can take it!

File: app.R [hunk 1]
```diff
@@ -1,30 +1,44 @@
...
 server <- function(input, output, session) {
     
+    output$sys_info <- renderTable({
+      df <- as_tibble(as.list(Sys.info()))
+      df <- as_tibble(cbind(Name = names(df), t(df)))
+      df <- df %>% rename(Value = V2)
+      df
+    })
+    
+    output$system_env <- renderTable({
```
Comment (aronatkins) [inline app.R:36]: I've used something like the following:

```r
ui <- shinyUI(fluidPage(
  h2("Environment variables"),
  tableOutput('environment')
))

server <- function(input, output, session) {
  output$environment <- renderTable({
    s <- Sys.getenv(names = TRUE)
    data.frame(name = names(s), value = as.character(s))
  })
}
```

**Comment by danielfrg** [comment]
Stop this madness

**Pull request merged**